opm/autodiff/SimulatorFullyImplicitBlackoilOutputEbos.hpp

```diff
@@ -0,0 +1,744 @@
+/*
```

Comment:
It would be nice if there was a lot more reuse (instead of copy&paste reimplementation) for the output. As it is now it is hard to maintain and at double the work.